src/main/java/org/junitpioneer/jupiter/RepeatFailedTestExtension.java

```diff
@@ -59,8 +60,8 @@ private static FailedTestRepeater repeaterFor(ExtensionContext context) {
 		Method repeatedTest = context.getRequiredTestMethod();
 		return context
 				.getStore(NAMESPACE)
-				.getOrComputeIfAbsent(repeatedTest.toString(), __ -> FailedTestRepeater.createFor(repeatedTest),
-					FailedTestRepeater.class);
+				.getOrComputeIfAbsent(repeatedTest.toString(),
+					defaultCreator -> FailedTestRepeater.createFor(repeatedTest), FailedTestRepeater.class);
```

Comment:
`__` means "nobody cares about this variable" - I want to keep using that.

Reply:
Never saw this before and was wondering that it's even allowed. Had to google it up to know that it's a way to show that the variable is not needed.

Reply:
adapted the regex to `^([a-z][a-zA-Z0-9]*|__)$`

Reply:
> adapted the regex to `^([a-z][a-zA-Z0-9]*|__)$`

As written only double underscore is not enough.

```
	public TempDirectory() {
		this((__, ___, dirPrefix) -> createDefaultTempDir(dirPrefix));
	}
``

Reply:
meh than i will add more than 2 _